helm/pinot/values.yaml

```diff
@@ -112,7 +112,7 @@ controller:
 
   jvmOpts: "-XX:ActiveProcessorCount=2 -Xms256M -Xmx1G -XX:+UseG1GC -XX:MaxGCPauseMillis=200 -Xlog:gc*:file=/opt/pinot/gc-pinot-controller.log"
 
-  log4j2ConfFile: /opt/pinot/etc/config/pinot-controller-log4j2.xml
+  log4j2ConfFile: /opt/pinot/etc/conf/pinot-controller-log4j2.xml
```

Comment:
which configuration stated it should be `conf` instead of `config` ?

Reply:
nvm i think it is from https://github.com/apache/pinot/blob/99d34e05e0fbe287df9fdabe1237b78459ae3e93/docker/images/pinot/Dockerfile#L45

Reply:
here is config located in image I found
https://github.com/apache/pinot/tree/master/docker/images/pinot/etc/conf

Reply:
I guess the configuration for conf path is from here 
https://github.com/apache/pinot/blob/master/pinot-distribution/pinot-assembly.xml#L243